autoload/merginal/buffers/historyLog.vim

```diff
@@ -12,8 +12,10 @@ function! s:f.generateHeader() dict abort
                 \ '']
 endfunction
 
+let g:merginal_log_commit_count = get(g:, 'merginal_log_commit_count', -20)
+
 function! s:f.generateBody() dict abort
-    let l:logLines = self.gitLines('log', '--format=%h %aN%n%ci%n%s%n', self.branch, '--')
+    let l:logLines = self.gitLines('log', g:merginal_log_commit_count,'--format=%h %aN%n%ai%n%s%n', self.branch)
```

Comment:
Why are you changing committer-time to author-time? I'm not going to debate which one is superior, but if you want such a change it'll need to be a different PR and it'll need to be configurable(because I prefer commiter-time)